objects/input-object-array/index.html

```diff
@@ -12,8 +12,8 @@ <h1>Notas de Alunos</h1>
     <input type="text" class="nome"></input><br></br>
     <label>NOTAS:</label>
     <input type="number" class="nota"></input><br></br>
-    <button type="submit" onclick="add()">Adicionar</button>
-    <div class="resultado"></div>
+    <button type="submit" onclick="handleSubmit()">Adicionar</button>
```

Comment:
```suggestion
    <button type="submit">Adicionar</button>
```
Também referente ao tópico 1.

Reply:
Fiz a correção Jv :koala:
commit: db923b361adb1349775c23d146962c1ed7553e17